src/nls/ja/strings.json

```diff
@@ -237,7 +237,7 @@
         "SECTION_COLLAPSE": ": ダブルクリックして折りたたむ",
         "SECTION_EXPAND": ": ダブルクリックして展開",
         "PANEL_COLUMN_COLLAPSE": ": クリックして折りたたむ",
-        "PANEL_COLUMN_EXPAND": ": クリックして展開",
+        "PANEL_COLUMN_EXPAND": " : クリックして展開",
```

Comment:
The `:` should be in the string. It's so that we can concatenate two strings and get something like `Title: Header`.